docs/tutorial/01-getting-started.md

```diff
@@ -0,0 +1,367 @@
+- Check **Allow connecting to non-HTTPS networks**
+
+> ℹ️ The 🌐 icon in the extension lets you switch back and forth between this Local network as well as test and main net.
+
+Now click the dApp's "Connect" button and follow the prompts to let the application communicate with Freighter. If it's successful, you should see your account info in the header along with a new "Fund Account" button and a tag for the current network. Click the "Fund Account" button so we can test some transactions.
+
+Once your wallet balance has some XLM, you should see the "GuessTheNumber" component update with a text box. Go ahead and enter some guesses. Right out of the gate we have nice UI to invoke methods on the contract.
+
+## 🔎 Understand the Contract Code
+
+Now navigate to our contract explorer. Click the "</> Debugger" button in the header. These are our contract developer tools. They'll let you explore the contracts available to your application, view documentation, and even run methods to help debug them right from your app!
+
+Select the `guess_the_number` contract and you should see its Contract ID from the local network deployment. You'll also see the contract's documentation for methods like:
+
+- `reset`: Update the number. Only callable by admin.
+- `guess`: Guess a number between 1 and 10
+
+This is coming directly from our contract's documentation. Let's open up the initial smart contract code in `contracts/guess-the-number/src/lib.rs` and walk through it.
+
+```rust
+#![no_std]
+use soroban_sdk::{contract, contractimpl, symbol_short, Address, BytesN, Env, Symbol};
+```
+
+Rust has a great [standard library](https://doc.rust-lang.org/std/) of types, functions, and other abstractions. But our smart contract will run in a constrained WebAssembly environment where the full library isn't available or needed. The `#![no_std]` directive forces us to use only core Rust features.
+
+We can still use explicitly imported features, though, and that's what the next line is doing. Here we're importing some essential items from Stellar's Soroban SDK, and we'll explain each as we get to them. You'll see that many of them replace items from the standard library but are designed for use in Soroban's environment. And the first is `contract`:
+
+```rust
+#[contract]
+pub struct GuessTheNumber;
+```
+
+The `#[...]` syntax in Rust is called an [attribute](https://doc.rust-lang.org/reference/attributes.html). It's a way to label code for the compiler to handle it with special instructions. *Inner* attributes (with the `#!`) apply to the scope they're within (meaning `!#[no_std]` applies to the whole file/module), and *Outer* attributes (just the `#`) apply to the next line.
+
+Here we're defining a [struct](https://doc.rust-lang.org/book/ch05-01-defining-structs.html) (a "structure" to hold values) and applying attributes of a Stellar smart contract.
+
+```rust
+const THE_NUMBER: Symbol = symbol_short!("n");
+pub const ADMIN_KEY: &Symbol = &symbol_short!("ADMIN");
```

Comment:
While I always prefer `&` here since you always refer to them as references, I say we stick with one or the other. Perhaps the value as to not introduce references at this point.